sdk/src/main/java/com/hedera/hashgraph/sdk/TokenCreateTransaction.java

```diff
@@ -190,7 +191,10 @@ public class TokenCreateTransaction extends Transaction<TokenCreateTransaction>
      * You must set the token supply type to FINITE if you set this field.
      */
     private long maxSupply = 0;
-
+    /**
+     * Metadata of the created token definition.
+     */
+    private byte[] tokenMetadata = {};
```

Comment:
I made it like this, because of the other fields in this class, like `tokenName`, `tokenSymbol`, `tokenMemo` (thought it would make sense to consistently name the field for the metadata as well).